Part 1: 
1. "values added: 20", no errors
2. "final result: 20", no errors
3. "values added: 20", no errors
4. Error, as result cannot be accessed outside the code block it is in bc it is declared with let
5. Error, result was declared with const and cannot be changed
6. Error, the call in line 13 is outside of result's code block, additinally there would be an error for the reason above
